package.json

```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "@calblueprint/eslint-config-react": "^0.0.3",
+    "@types/styled-components": "^5.1.29",
```

Comment:
Do we still need the `@types/styled-components` package? Not sure if the error showed up again..

Reply:
When she got rid of it, the PR failed the TypeScript compiler Github action :(( seems like we might need it!

Reply:
rip I'll sort thru the eslint package stuff